### DISCUSSION TITLE
how to disable console output "{result_flag: True}"

### DISCUSSION THREAD

**Discussion by Holy-Shine** [fastapi/fastapi#6181] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [x] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI


app = FastAPI()
@app.get('/hello')
async def hello():
    return "hello"
```

### Description

Every time I make a http request to my fastapi server, I get an ouput like this in my consonle: 
`{result_flag}: True`
How to disable this log?

### Operating System

Linux

### FastAPI Version

1.0.3

### Python Version

3.9.5

**Comment by tiangolo** [comment]
Are you sure you are getting that log with that sample app?

It seems like you would have something else interacting and generating that log, not FastAPI.

**Comment by Holy-Shine** [comment]
yes.
with unvicorn runner:
```
uvicorn.run(app, port=8001, host='0.0.0.0', log_level=logging.WARNING)
```

**Comment by Holy-Shine** [comment]
output disappear after change my dev environment, I'll checkout if other module do output
